standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -668,6 +671,20 @@ public void rollbackTransaction() {
     }
   }
 
+  private void setTransactionSavePoint(String savePoint) {
+    if (savePoint != null) {
+      ExecutionContext ec = ((JDOPersistenceManager) pm).getExecutionContext();
+      ec.getStoreManager().getConnectionManager().getConnection(ec);
```

Comment:
Should we try/catch null exception incase any of these becomes null?

Reply:
I believe it is not required to check against null. These instances are created with the pm instance and cannot be null.

Reply:
Ack